libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/puck/LocationPuckOptions.kt

```diff
@@ -95,20 +92,23 @@ class LocationPuckOptions private constructor(
      */
     class Builder(private val context: Context) {
 
-        private var freeDrivePuck: LocationPuck? = null
-        private var destinationPreviewPuck: LocationPuck? = null
-        private var routePreviewPuck: LocationPuck? = null
-        private var activeNavigationPuck: LocationPuck? = null
-        private var arrivalPuck: LocationPuck? = null
-        private var defaultPuck: LocationPuck? = null
+        private var freeDrivePuck: LocationPuck = regularPuck(context)
+        private var destinationPreviewPuck: LocationPuck = regularPuck(context)
+        private var routePreviewPuck: LocationPuck = regularPuck(context)
+        private var activeNavigationPuck: LocationPuck = navigationPuck(context)
+        private var arrivalPuck: LocationPuck = navigationPuck(context)
```

Comment:
Should we use navigation or regular puck in arrival state? With the previous implementation you used regular puck, but now it is navigation one.

Reply:
> Please update current.txt files.

I did run `make ui-update-api`. No changes took effect, because `defaultPuck` is still exposed as a function.

Reply:
> With the previous implementation you used regular puck, but now it is navigation one.

That was a mistake. We are using `navigationPuck` for arrival state.

@Zayankovsky